step-release-vis/src/app/components/grid/canvas-grid/canvas-grid.component.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
the .spec in the filename is a bit confusing, also having - and . as separator in the middle of the filename a bit.. 
- is bad because some g3 infra ill not like it later on, I would prefer _ instead.
the . should be only before the extension at the end, and not at the middle, using just snake_case.ext or CamelCase.ext for the whole filename everywhere would be better.

Reply:
It's the default way in Angular though. The CLI command `ng generate component CanvasGrid` creates 4 files with this naming. Should I rename all the the files in the project?

Reply:
Could check what convention is used in Angular projects in g3?

Reply:
ack, I will check, if we have any pointers on this

Reply:
Renamed the files. However, if I rename the app.module.ts file, the CLI can't recognize the NgModule and shows errors. So I left the app.* as thery were, but renamed the custom ones.